.github/CONTRIBUTING.md

```diff
@@ -394,6 +394,12 @@ yarn load:env ./scripts/seed-lightdash.sh
 
 # run
 yarn load:env yarn dev
+
+# Log in dev mode
+When navigating to http://localhost:3000 you will be prompt to the login page, you can use our demo login details:

```

Comment:
This line is not a valid `Bash` command, so I'd make this a comment (prefix with #)

Reply:
ahh, you merged, never mind. It is a minor thing

Reply:
I copied the same way that it was on the using-docker instructions, can change these on a separate PR tho 👍  waht do you think @rephus ?

Reply:
It is ok as it is 👍